packages/jest-config/src/normalize.ts

```diff
@@ -594,6 +595,17 @@ export default function normalize(
             });
         }
         break;
+      case 'testSequencer':
+        {
+          const option = oldOptions[key];
+          value =
+            option &&
+            getSequencer(newOptions.resolver, {
```

Comment:
you need to add a dependency on `@jest/test-scheduler` to `jest-config` so it works in yarn PnP

Reply:
I don't think it really need a dependency on `@jest/test-sequencer`.
In `jest-config`, it only parse `testSequencer` to package name or path, not import it.

Reply:
It should resolve to an absolute path, and for that to happen reliably without a `node_modules` directory you need to add the dependency. the `require` itself will happen in `jest-core`, but `config.testSequencer` should be an absolute path when coming out of `jest-config`.

That's the way all other things work as well, when Jest bundles a default implementation for something (`jest-jasmine`, `babel-jest`, `jest-environment-jsdom` etc)

Reply:
Since `require.resolve` does the same resolution as `require`, I agree this needs a dependency

Reply:
BTW, maybe we should add `jest-runner` to `jest-config` dependency.
It also use same way.